auth-web/src/locales/en.json

```diff
@@ -1,10 +1,20 @@
 {
   "message": "hello i18n !!",
   "loginFailedMessage": "Incorrect Incorporation Number or Passcode",
-  "techSupportPhone": "250 387-7848",
+  "labelTollFree": "Toll Free:",
+  "labelVictoriaOffice": "Victoria Office:",
+  "labelPhone": "Phone:",
+  "labelEmail": "Email:",
+  "techSupportPhone": "250-387-7848",
   "techSupportTollFree": "1-877-526-1526",
-  "techSupportEmail": "bcregistries@gov.bc.ca",
+  "techSupportEmail": "BCRegistries@gov.bc.ca",
   "techSupportEmailSubject": "Business Registry Support Request",
+  "maximusSupportPhone": "1-250-370-1033",
+  "maximusSupportTollFree": "1-877-370-1033",
+  "maximusSupportEmail": "BCRegistries@gov.bc.ca",
+  "maximusSupportEmailSubject": "BC Registries and Online Services - Support Request",
```

Comment:
There's a new set of key-values here because the Maximus Tier 1 help desk supports names and coops/corps teams at the moment -- they don't support BC Online (yet) or PPR/MH.